### ISSUE TITLE
Account recovery request

### ISSUE THREAD

**Issue by msliberator** [pypi/support#4559]
### PyPI Username

msliberator

### Reason for request

My phone will not startup and I don't have access to my 2FA authentication app because it is on my phone. I have already tried using my recovery codes but all come back as invalid. Do Recovery codes expire? 

### Recovery Codes

- [X] I never generated or have lost access to the recovery codes for my account

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

### Acknowledgement

- [X] I understand that it may take a significant amount of time to process my account recovery request.

**Comment by Thespi-Brain** [comment]
Hi @msliberator,

Recovery codes typically don't expire. I took a look in your account and your original recovery codes were still remaining. But, just to give you an opportunity to set up 2FA on a new device and as a fresh start, I have reset your 2FA and a password reset has been issued. You should be able to regain access to your account after completing the password reset. Let us know if you have any more issues. I will keep this issue open for another week unless I hear from you earlier. Thanks.

**Comment by msliberator** [comment]
No, thank you @Thespi-Brain, you've helped me when the recovery keys did not. I'm in my account again! Completed password reset and setup F2A, logged out and in again to check and everything looks okay. 

Thank you again.

**Comment by Thespi-Brain** [comment]
Closing as resolved!
